Source/utils/display.cpp

```diff
@@ -424,7 +475,7 @@ bool SpawnWindow(const char *lpWindowName)
 	}
 #endif
 

```

Comment:
We probably want to have the enter key act as submit rather then line break:
```suggestion

#ifdef USE_SDL3
	SDL_SetHint(SDL_RETURN_KEY_HIDES_IME, "1");
#endif
```